README.md

```diff
@@ -17,7 +17,7 @@ Currently, AutoGluon-Cloud supports [AWS SageMaker](https://aws.amazon.com/sagem
 # First install package from terminal:
 # pip install -U pip
 # pip install -U setuptools wheel
-# pip install --pre autogluon.cloud  # You don't need to install autogluon itself locally
+# pip install autogluon.cloud==0.2.0  # You don't need to install autogluon itself locally
```

Comment:
Why specify the specific version?

Reply:
Trying to follow what autogluon has: https://github.com/autogluon/autogluon/blob/master/README.md

Reply:
AutoGluon has `# pip install autogluon  # autogluon==0.7.0` which is also a bit weird